app/models/user.rb

```diff
@@ -66,6 +94,10 @@ def generate_activation_token!
     retry
   end
 
+  def invalidate_reset_token!
+    update! reset_sent_at: nil, reset_digest: nil # Remove expired/valid tokens.
+  end
+
```

Comment:
Unless this needs to be reused, we can just move this logic into the method above

Reply:
It's  reused in `reset_password_controller#reset` action.